whipper/test/test_common_config.py

```diff
@@ -66,3 +66,25 @@ def testDefeatsCache(self):
         defeats = self._config.getDefeatsCache(
             'PLEXTOR ', 'DVDR   PX-L890SA', '1.05')
         self.assertEquals(defeats, True)
+
+    def test_get_musicbrainz_server(self):
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          'musicbrainz.org')
+
+        self._config._parser.add_section('musicbrainz')
+
+        self._config._parser.set('musicbrainz', 'server',
+                                 '192.168.2.141:5000')
+        self._config.write()
+
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          '192.168.2.141:5000')
```

Comment:
I would have these calls listed without empty lines between them, and preferably with a `#comment` just prior to them to give a more "prosaic" version of what's going on.

Reply:
I think it might even be nicer to make the "prosaic" comments the `msg=` param to `assertEquals()`, as mentioned below.